frontend/src/scenes/session-recordings/player/Time.tsx

```diff
@@ -32,7 +32,12 @@ export function SeekBack({ style, sessionRecordingId, playerKey }: TimeControlPr
         <Tooltip
             placement="top"
             overlayInnerStyle={{ minHeight: 'auto' }}
-            overlay={`Back ${jumpTimeMs / 1000}s (← left arrow)`}
+            overlay={
+                <>
+                    Back {jumpTimeMs / 1000}s (← left arrow) <br />
+                    (Hold <kbd>alt</kbd> to move one frame at a time)
+                </>
+            }
```

Comment:
Could there be a clearer way to surface this? When i first saw this, I was trying to hold down alt and click the seek icons, which triggered the seek X s instead of seek single frame code. 

An easy solution here could be to also listen for if alt is held down while the icons are clicked.

Reply:
Nit: change `alt` to `alt/option` to be platform agnostic

Reply:
Yeah I wondered about this as well. I guess we could listen for alt being pressed and change the content of the button to something clearer (maybe the number 1 is enough). 

Generally these key commands we need to surface somehow in a more discoverable way I think... But that might be a bigger efffort.